requirements-full.txt

```diff
@@ -247,37 +222,29 @@ terminado==0.12.1; python_version >= "3.6"
 testpath==0.5.0; python_version >= "3.7"
 textwrap3==0.9.2; python_version >= "3.6"
 thepassiveinvestor==1.0.10
-threadpoolctl==3.0.0; python_version >= "3.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.7"
-tokenize-rt==4.2.1; python_full_version >= "3.6.1"
-toml==0.10.2; python_full_version >= "3.6.2" and python_version >= "3.6" and (python_version >= "3.5" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" and python_version >= "3.5") and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" and python_version >= "3.6")
+threadpoolctl==3.0.0; python_version >= "3.7"
+toml==0.10.2; python_version >= "3.5" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" and python_version >= "3.5"
 tomli==1.2.2; python_full_version >= "3.6.2" and python_version >= "3.7"
 tornado==6.1; python_full_version >= "3.6.1" and python_version >= "3.7"
 tqdm==4.62.3; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6"
 tradingview-ta==3.2.9; python_version >= "3.6"
 traitlets==5.1.1; python_full_version >= "3.6.1" and python_version >= "3.7"
-types-python-dateutil==2.8.3
```

Comment:
I think you should re-add all of these below.

This may be the reason why the test is failing